### PR TITLE
WIP for case bundle certification process

### PR DESCRIPTION
**Pull request by bluzarraga**
Full list of linting rules and their descriptions for reference: https://github.ibm.com/IBMPrivateCloud/content-verification/tree/v2.1/docs/rules

### CODE REVIEW

File: deploy/crds/elasticstack.ibm.com_elasticstacks_crd.yaml [hunk 1]
```diff
@@ -15,7 +15,573 @@ spec:
   validation:
     openAPIV3Schema:
       type: object
-      x-kubernetes-preserve-unknown-fields: true
```
Comment (dian-xh) [inline deploy/crds/elasticstack.ibm.com_elasticstacks_crd.yaml:18]: used to cause issue before k8s 1.15 without this

File: deploy/crds/elasticstack.ibm.com_elasticstacks_crd.yaml [hunk 2]
```diff
@@ -15,7 +15,573 @@ spec:
...
+      description: MongoDB is the Schema for the mongodbs API
```
Comment (dian-xh) [inline deploy/crds/elasticstack.ibm.com_elasticstacks_crd.yaml:18]: we can use the description in the crd

File: deploy/olm-catalog/ibm-elastic-stack-operator/3.0.5/elasticstack.ibm.com_elasticstacks_crd.yaml [hunk 3]
```diff
@@ -15,7 +15,7 @@ spec:
   validation:
     openAPIV3Schema:
       type: object
-      x-kubernetes-preserve-unknown-fields: true
+      x-kubernetes-preserve-unknown-fields: true #this would need to be fleshed out the way deploy/crds/elasticstack.ibm.com_elasticstacks_crd.yaml was for ResourceShcemaIsValid
```
Comment (dian-xh) [inline deploy/olm-catalog/ibm-elastic-stack-operator/3.0.5/elasticstack.ibm.com_elasticstacks_crd.yaml:18]: we can leave this one alone if not blocking case bundle

File: deploy/crds/elasticstack.ibm.com_elasticstacks_crd.yaml [hunk 4]
```diff
@@ -15,7 +15,573 @@ spec:
...
+                    requests:
+                      type: object
+                      properties:
+                        memory:
+                          type: string
+                scope:
+                  type: object
+                  properties:
+                    namespaces: #both these values are null
+                      type: string
```
Comment (dian-xh) [inline deploy/crds/elasticstack.ibm.com_elasticstacks_crd.yaml:294]: this is a list

File: deploy/crds/elasticstack.ibm.com_elasticstacks_crd.yaml [hunk 5]
```diff
@@ -15,7 +15,573 @@ spec:
...
+                    nodes:
+                      type: string
```
Comment (dian-xh) [inline deploy/crds/elasticstack.ibm.com_elasticstacks_crd.yaml:296]: is is a map

File: deploy/crds/elasticstack.ibm.com_v1alpha1_elasticstack_cr.yaml [hunk 6]
```diff
@@ -116,9 +116,9 @@ spec:
       requests:
         memory: 64Mi
     scope:
-      namespaces: null
-      nodes: null
-    tolerations: null
+      namespaces: ""
```
Comment (dian-xh) [inline deploy/crds/elasticstack.ibm.com_v1alpha1_elasticstack_cr.yaml:119]: list

File: deploy/crds/elasticstack.ibm.com_v1alpha1_elasticstack_cr.yaml [hunk 7]
```diff
@@ -116,9 +116,9 @@ spec:
...
+      nodes: ""
```
Comment (dian-xh) [inline deploy/crds/elasticstack.ibm.com_v1alpha1_elasticstack_cr.yaml:120]: map

File: helm-charts/ibm-icplogging/ibm_cloud_pak/manifest.yaml [hunk 8]
```diff
@@ -0,0 +1,159 @@
...
+      os: "linux"
+      arch: "ppc64le"
+  - repository: icp-initcontainer:1.0.0-build.3
+    pull-repository: "quay.io/opencloudio/icp-initcontainer:1.0.0-build.3"
+    platform:
+      os: "linux"
+      arch: "s390x"
+- image: icp-elasticsearch-oss:6.6.1-build.1
+  references:
+  - repository: cp-elasticsearch-oss:6.6.1-build.1
```
Comment (dian-xh) [inline helm-charts/ibm-icplogging/ibm_cloud_pak/manifest.yaml:60]: icp

File: helm-charts/ibm-icplogging/templates/ls-deployment.yaml [hunk 9]
```diff
@@ -13,7 +13,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 {{ template "elk.settings.all.prepare" . -}}
-{{ template "apiVersionTryV1" .  }}
+{{ template "apiVersionTryV1" .  }} #tagged for NoDeprecatedAPIs (v1beta2), not really sure what this template value is
```
Comment (dian-xh) [inline helm-charts/ibm-icplogging/templates/ls-deployment.yaml:16]: this template conditionally renders at the api level that the underlying k8s environment supports

**Comment by bluzarraga** [comment]
closing in favor of https://github.com/IBM/ibm-elastic-stack-operator/pull/25

**Pull request closed without merging**